rllib/models/tf/layers/relative_multi_head_attention.py

```diff
@@ -121,3 +125,14 @@ def rel_shift(x: TensorType) -> TensorType:
         x = tf.reshape(x, x_size)
 
         return x
+
+
+class PositionalEmbedding(tf.keras.layers.Layer):
```

Comment:
Add comments on what this does (how it initializes embedding per position based on cos/sin something)